tests/unit/cli/storage_layout/test_storage_layout_overrides.py

```diff
@@ -26,6 +28,31 @@ def test_storage_layout_overrides():
     assert out["layout"] == expected_output
 
 
+def test_storage_layout_overrides_json():
+    code = """
+a: uint256
+b: uint256"""
+
+    storage_layout_overrides = {
+        "a": {"type": "uint256", "slot": 1, "n_slots": 1},
```

Comment:
let's move the slot so we don't accidentally pass if we just reorder the variables in the code
```suggestion
        "a": {"type": "uint256", "slot": 5, "n_slots": 1},
```